js/tweek-client/src/TweekClient.ts

```diff
@@ -23,20 +23,8 @@ export default class TweekClient implements ITweekClient {
     }
   }
 
-  private fetchChunk<T>(path: string, _config?: FetchConfig): Promise<T> {
-    const {
-      casing,
-      flatten,
-      baseServiceUrl,
-      convertTyping,
-      context,
-      include,
-      ignoreKeyTypes,
-      onError,
-    } = <TweekInitConfig & FetchConfig>{
-      ...this.config,
-      ..._config,
-    };
+  private fetchChunk<T>(path: string, _config: TweekInitConfig): Promise<T> {
```

Comment:
change _config type to TweekInitConfig & FetchConfig to clarify that it's an amalgamation of the two configs